src/main/java/dl/news/portal/web/config/Application.java

```diff
@@ -2,8 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @SpringBootApplication(scanBasePackages = "dl.news.portal")
+@EntityScan("dl.news.portal.domain.entity")
+@EnableJpaRepositories(basePackages = "dl.news.portal.domain.repository")
```

Comment:
Why do you need this 

> @EntityScan("dl.news.portal.domain.entity")
@EnableJpaRepositories(basePackages = "dl.news.portal.domain.repository")

It's redundant

Reply:
I did use it because this class placed deeper in hierarchy.

Reply:
Change you package hierarchy to the correct form and remove this annotations

Reply:
Done